# Description: Get the value of a GenICam parameter.

# Name of the parameter
string name

---

# Value of the parameter converted to a string
string value

# Return code
rc_genicam_camera/ReturnValue return_code
